src/models/linear_network.py

```diff
@@ -230,6 +244,8 @@ def forward(self, x, pred_month=None):
         # flatten
         x = x.contiguous().view(x.shape[0], -1)
         if self.include_pred_month:
+            # @gabrieltseng HELP!
```

Comment:
See the comment above; I think this is why we should have seperate arrays returned for the prediction month and the nowcasting stuff